### ISSUE TITLE
Save pulseaudio inputs/outputs

### ISSUE THREAD

**Issue by tobiasBora** [Houston4444/RaySession#131]
In pipewire, it's possible to change the ALSA source of some programs directly using pulseaudio (I don't think this can be done with qjackctl directly). However, this is not saved by RaySession.

To get an example of use: https://github.com/BespokeSynth/BespokeSynth/issues/500#issuecomment-954664962

EDItT: apparently scripts can be practical to do that task (thanks LAM) http://raysession.tuxfamily.org/en/manual#session_script but I guess it can still be nice if it's automatically detected.

**Comment by Houston4444** [comment]
Mmmh... The scripted template "with JACK configuration memory" also saves pulseaudio bridges, but I am (pretty) sure this scripts wont work with pipewire. I think you could be able to achieve thé behavior you except with session scripts, with little work, let me know if you achieve this. 
I note with tour screenshot that I have improves to do with patchbay and pipewire: sooperlooper Alsa midi port grouped with MidiBridge (instead sooperlooper), and toi Big boxes because pw generates verrry long group names.

**Comment by Houston4444** [comment]
> I guess it can still be nice if it's automatically detected.

No, RS basically doesn't manage any hardware config, and it is Well this way, for session portability
